test/evp_extra_test.c

```diff
@@ -767,6 +767,40 @@ static int test_EVP_PKCS82PKEY(void)
 }
 #endif
 
+/* This uses kExampleRSAKeyDER and kExampleRSAKeyPKCS8 to verify encoding */
+static int test_privatekey_to_pkcs8(void)
+{
+    EVP_PKEY *pkey;
```

Comment:
Travis failure here. Set this to NULL to avoid unitialised use on an error path

Reply:
Fixed